### DISCUSSION TITLE
SQL Error  after moving wikiJS database

### DISCUSSION THREAD

**Discussion by omougel** [requarks/wiki#6786] [Help / Questions] [answered]
HI,
I've got some issues after moving my wikijs 2.5 postgresql database from a server to another one. (using pg_dump and pg_restore)
The whole  new wiki works fine but when i try to save some updates of a page, I receive this infamous message :

insert into "pageHistory" ("action", "authorId", "content", "contentType", "createdAt", "description", "editorKey", "hash", "isPrivate", "isPublished", "localeCode", "pageId", "path", "publishEndDate", "publishStartDate", "title", "versionDate") values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17) returning "id" - une valeur NULL viole la contrainte NOT NULL de la colonne « id »

I understand : WikiJS doesn't provide a value to an id field, maybe due to a trigger issue but I'm not sure.

Someone has seen and resolved this ?
Any ideas ?
Thank's

**Comment by NGPixel** [comment, marked as answer]
The id column is an autoincrement and generates an id by itself. So your dump / restore is missing that instruction.

**Reply by omougel** [reply]
Hi
I have found the reason why. The sequences were missing ! 
Thank's for this great wiki you create.

**Comment by nik-weter** [comment]
Hello. My situation's the same.
But i dont know who and when move database. WIKIJS just have worked fore 2 years, and then i decided edit some page and get this error.
Someone moved wiki database and then have read for 2 years without editing.
There is some way to make my wiki work again? Reinitialyse database? Migratin? Export to new instance?
